CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Guidelines for Contribution
+
+## How can I contribute to vscode-seneca-college project?
+
+Never contributed to open source before? Are you curious about how contributions function in our project?
+Here is a brief summary!
+
+**Here is a brief overview of how to contribute to vscode-seneca-college:**
+
+---
+
+1. Start by identifying an issue or a feature that you would want to add. Choose a issue from the issues tab that you would like to work on. Feel free to open a new issue in the project if it is not in the list of issues.
```

Comment:
If I am being REALLY nit picky, the second sentence should also be *an* issue instead of *a* issue.